src/components/todo-item.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <li>
     <div class="item-container" v-if="!isEditing">
-      <input class="toggle" type="checkbox" v-model="todo.completed"/>
+      <input @click="toggleStatus" class="toggle" type="checkbox" v-model="todo.completed"/>
       <label>
         {{todo.title}}
       </label>
       <button @click="editMode" class="edit-item">Edit</button>
-      <button @click="$emit('removeTodoItem')" class="remove-item">Remove</button>
+      <button @click="removeTodoById" class="remove-item">Remove</button>
```

Comment:
using naming `removeTodoById ` you need to pass `id` as function param. better to change `removeTodo` in this case

Reply:
done.